### PR TITLE
Add Header / home pages

### PR DESCRIPTION
**Pull request by kkshen3352**
add header rwd

### CODE REVIEW

**Comment by ClarityAcuity** [comment]
https://stackoverflow.com/questions/1889559/git-diff-to-ignore-m

**Comment by kkshen3352** [comment]
CRLF change LF
# Remove everything from the index
$ git rm --cached -r .

# Re-add all the deleted files to the index
# You should get lots of messages like: "warning: CRLF will be replaced by LF in <file>."
$ git diff --cached --name-only -z | xargs -0 git add

# Commit
$ git commit -m "Fix CRLF"

File: src/components/footer.js [hunk 1]
```diff
@@ -2,11 +2,12 @@ import React from "react"
...
 
 const Footer = () => (
-  <Flex bg="blue.50" align="center" justify="center">
+  <Flex bg="black.000" align="center" justify="center"
+  color="white" h="100px" position="relative" pb="0">
     <footer>
       © {new Date().getFullYear()}, Built with
       {` `}
-      <a href="https://www.gatsbyjs.org">Gatsby</a>
+      <a href="https://www.gatsbyjs.org">關於我</a>
```
Comment (ClarityAcuity) [inline src/components/footer.js]: 不要動 build with Gatsby
關於我分開寫

File: src/components/header.js [hunk 2]
```diff
@@ -1,90 +1,76 @@
...
-          display={{ base: show ? "block" : "none", lg: "flex" }}
-          mt={{ base: 4, lg: 0 }}
-          alignItems={{ lg: "center" }}
+        </Heading>
+        <Box display={{ base: "none", lg: "flex" }}>
+        <Box margin="1rem"><Link to="/">首頁</Link></Box>
+        <Box margin="1rem"><Link to="/demo">演示</Link></Box>
+        <Box margin="1rem"><Link to="/about">關於我</Link></Box>
+        
+        </Box>
```
Comment (ClarityAcuity) [inline src/components/header.js]: 建立 link 物件
[link1, link2, ...].map() 畫

File: src/components/header.js [hunk 3]
```diff
@@ -1,90 +1,76 @@
...
+        
+        <Box 
+        display={{ base: "block", lg: "none" }} 
+        margin="1rem"
+        position="absolute"
+        right="0"
```
Comment (ClarityAcuity) [inline src/components/header.js]: 與上面 box (wrapper) 整合用一個
內容 (children) 隨尺寸變化

File: src/pages/posts/highlight-test.md [hunk 4]
```diff
@@ -1,3 +1,100 @@
+<<<<<<< HEAD
```
Comment (ClarityAcuity) [inline src/pages/posts/highlight-test.md]: 不要動

File: src/pages/index.js [hunk 5]
```diff
@@ -66,6 +66,7 @@ function IndexPage({ data }) {
       <Link to="/rect-gl">Go to rect GL page</Link>
       <Link to="/ray-tracing">Go to ray tracing page</Link>
     </Layout>
+
```
Comment (ClarityAcuity) [inline src/pages/index.js]: 刪除

File: src/pages/posts/js-questions.mdx [hunk 6]
```diff
@@ -1,3 +1,8 @@
+<<<<<<< HEAD
```
Comment (ClarityAcuity) [inline src/pages/posts/js-questions.mdx]: 不要動

**Review by ClarityAcuity — COMMENTED** [review]
用 prettier 統一排版

File: src/components/header.js [hunk 7]
```diff
@@ -1,90 +1,82 @@
...
-              xmlns="http://www.w3.org/2000/svg"
-            >
-              <title>Menu</title>
-              <path d="M0 3h20v2H0V3zm0 6h20v2H0V9zm0 6h20v2H0v-2z" />
-            </svg>
+            </Box>
+          </Heading>
+          <Box display={{ base: "none", lg: "flex" }}>
+            {Links.map(Links => (
+              <Box margin="1rem">
```
Comment (ClarityAcuity) [inline src/components/header.js]: { name, path }

File: src/components/header.js [hunk 8]
```diff
@@ -1,90 +1,82 @@
...
+                position="absolute"
+                right="0"
+                as={IconButton}
+                aria-label="Options"
+                icon={<HamburgerIcon />}
+                size="lg"
+                variant="outline"
+              />
+              <MenuList color="black">
+                {Links.map(Links => (
```
Comment (ClarityAcuity) [inline src/components/header.js]: { ... }

File: src/components/layout/style.css [hunk 9]
```diff
@@ -0,0 +1,622 @@
+html {
```
Comment (ClarityAcuity) [inline src/components/layout/style.css]: 移除

**Review by ClarityAcuity — APPROVED** [review]

**Pull request merged**